brands/shop/car.json

```diff
@@ -324,7 +324,10 @@
     "tags": {
       "brand": "Автомир",
       "name": "Автомир",
-      "shop": "car"
+      "name:en": "Autoworld",
+      "shop": "car",
+      "brand:wikidata": "Q4056321",
+      "brand:wikipedia": "ru:Автомир (автодилер)",
```

Comment:
This last line should not have a comma.  I think you'll be set after that.

Reply:
LOL thanks!